website/members/templates/members/email/yearly_study_check_reminder.html

```diff
@@ -0,0 +1,14 @@
+{% extends "email/html_email.html" %}
+
+{% block content %}
+  Dear {{ name }},<br>
+
+  <p>You have a study membership with Thalia. This means that you'll be a member as long as you're enrolled in either a Computing Science or an Information Sciences programme.</p>
```

Comment:
Deze regel had je aangepast (verbeterd) voor deze template maar staat ook in de andere emails nog op de oude manier dus die kunnen ook wel dezelfde aanpassing gebruiken

Reply:
Ah ja goeie